src/components/language/dto/language.dto.ts

```diff
@@ -155,6 +155,9 @@ export class Language extends Resource {
   @Field()
   readonly hasExternalFirstScripture: SecuredBoolean;
 
+  // Internal First Scripture == true via this engagement
+  readonly firstScriptureEngagement?: ID;
```

Comment:
does this need to be a `Secured<ID>`? I was noticing that `securedProps` in the languageService doesn't pick this up as being secured.

Reply:
I was just basing it on the `hasExternalFirstScripture`. Since this is based on that and one other thing, it felt like it was sufficient.